HTTPService/pkg/entities/requests.go

```diff
@@ -0,0 +1,14 @@
+package entities
+
+type GetUserReq struct {
+	Id string
+}
+
```

Comment:
this is the correct way, and remeber  separate the grpc and hptt structs (resp and req) they can get in conflic